src/main/java/no/nav/pto/veilarbportefolje/aktiviteter/AktivitetService.java

```diff
@@ -0,0 +1,91 @@
+package no.nav.pto.veilarbportefolje.aktiviteter;
+
+import lombok.extern.slf4j.Slf4j;
+import no.nav.pto.veilarbportefolje.database.PersistentOppdatering;
+import no.nav.pto.veilarbportefolje.domene.AktoerId;
+import no.nav.pto.veilarbportefolje.service.PersonIdService;
+import no.nav.pto.veilarbportefolje.util.BatchConsumer;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+
+import static io.vavr.control.Try.run;
+import static no.nav.pto.veilarbportefolje.util.BatchConsumer.batchConsumer;
+
+@Slf4j
+@Service
+public class AktivitetService {
+
+    private final PersonIdService personIdService;
+    private final AktivitetDAO aktivitetDAO;
+    private final PersistentOppdatering persistentOppdatering;
+
+    @Autowired
```

Comment:
this constructor can be replaced with Lombok annotation: @RequiredArgsConstructor